app/src/vuex/modules/session.js

```diff
@@ -281,10 +331,24 @@ export default ({ apollo }) => {
           title,
           address: JSON.parse(sessionEntry).address,
           sessionType: JSON.parse(sessionEntry).sessionType,
+          accountType: JSON.parse(sessionEntry).accountType,
         })
       })
       return allSessionAddresses
     },
+    getCurrentAccountType({
+      rootState: {
+        connection: { networks, network },
+      },
+    }) {
+      const currentNetwork = networks.find(({ id }) => id === network)
+      const sessionEntry = localStorage.getItem(`session_${currentNetwork.id}`)
+      if (!sessionEntry)
+        return currentNetwork.network_type === "cosmos"
```

Comment:
how about a `currentNetwork.defaultAccountType`?

Reply:
I deleted this function. With the mutation is enough